ass_man/import_manager.py

```diff
@@ -38,7 +38,8 @@ def import_model_file(request):
             if row['memory']:
                 new_model.memory = row['memory']
             nps = []
-            for i in range(1, int(row['network_ports']) + 1):
+            csv_num_netports = int(row['network_ports']) if row['network_ports'] else 0
+            for i in range(1, csv_num_netports + 1):
```

Comment:
this is the same as desired behavior for if pp column is blank in the file? i think any column that holds an optional value is fair game to be left blank- do we currently handle that all?